src/test/java/com/puppycrawl/tools/checkstyle/gui/ParseTreeTablePModelTest.java

```diff
@@ -163,8 +163,109 @@ public void testJavadocChild() {
         Assert.assertTrue(javadoc instanceof DetailNode);
         Assert.assertEquals(JavadocTokenTypes.JAVADOC, ((DetailNode) javadoc).getType());
         final Object javadocChild = parseTree.getChild(javadoc, 2);
-        Assert.assertTrue(javadoc instanceof DetailNode);
+        Assert.assertTrue(javadocChild instanceof DetailNode);
         Assert.assertEquals(JavadocTokenTypes.TEXT, ((DetailNode) javadocChild).getType());
     }
 
+    @Test
+    public void testGetIndexOfChild() {
+        final DetailAST nChild = tree.getFirstChild().getNextSibling().getNextSibling();
+        final ParseTreeTablePModel parseTree = new ParseTreeTablePModel(null);
+        final int index = parseTree.getIndexOfChild(tree, nChild);
+        Assert.assertEquals(2, index);
+    }
+
+    @Test
+    public void testGetValueAt() {
+        final DetailAST node = tree.getFirstChild()
+            .getNextSibling()
+            .getNextSibling()
+            .getNextSibling();
+
+        final ParseTreeTablePModel parseTree = new ParseTreeTablePModel(null);
+        final Object treeModel = parseTree.getValueAt(node, 0);
+        final String type = (String) parseTree.getValueAt(node, 1);
+        final int line = (int) parseTree.getValueAt(node, 2);
+        final int column = (int) parseTree.getValueAt(node, 3);
+        final String text = (String) parseTree.getValueAt(node, 4);
+
+        Assert.assertEquals("IDENT", type);
+        Assert.assertEquals(4, line);
+        Assert.assertEquals(6, column);
+        Assert.assertEquals("InputJavadocAttributesAndMethods", text);
+        Assert.assertNull(treeModel);
+
+        try {
+            parseTree.getValueAt(node, 231);
+            Assert.fail("IllegalStateException expected");
+        }
+        catch (IllegalStateException ex) {
+            Assert.assertEquals("Unknown column", ex.getMessage());
+        }
+
+    }
+
+    @Test
+    public void testGetValueAtDetailNode() {
+        final DetailAST commentContentNode = tree.getFirstChild().getNextSibling().getFirstChild();
```

Comment:
I have added the requested test logging to indicate the intended nodes within the test file parse tree models. This node will be the child comment node of a /* node, indicating a Javadoc.